packages/devtools_app/lib/src/screens/debugger/debugger_screen.dart

```diff
@@ -383,10 +385,10 @@ class _DebuggerStatusState extends State<DebuggerStatus> with AutoDisposeMixin {
       return 'paused$reason';
     }
 
-    final fileName = ' at ' + frame.location.script.uri.split('/').last;
-    final script = await scriptManager.getScript(frame.location.script);
+    final fileName = ' at ' + frame.location!.script!.uri!.split('/').last;
+    final script = await scriptManager.getScript(frame.location!.script!);
     final pos =
-        SourcePosition.calculatePosition(script, frame.location.tokenPos);
+        SourcePosition.calculatePosition(script, frame.location!.tokenPos!);
```

Comment:
pull `frame.location!` out into a local var and then use that local to create another local var `location.script!`. Then use in these three lines to avoid the redundant null checks